### PR TITLE
DOC: Revise README to reStructuredText for PyPi

### PR DESCRIPTION
**Pull request by JDWarner**
* Convert from Markdown to reStructuredText for PyPi rendering
* Remove part of the initial blurb pertaining to the SciPy talk
* Add minimal installation instructions including `matplotlib` dependency
  * Remove NumPy dependency in `setup.py`
* Use GitHub's cloud hosting to show expected result of example code in the README without expanding package size

You can view the result of these changes on GitHub here: https://github.com/jdwarner/trendvis/tree/readme_rst

Example 1
ex0

Example 2
ex1

Example 3
(snipped, see below)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        twin0, yvals+0.075, yvals-0.1, edgecolor='none', color='darkorange')
+
+    ex2.axes[2][1].fill_between(
+        twin0, yvals+0.075, yvals-0.1, edgecolor='none', color='darkorange')
+
+    ex2.axes[2][2].fill_between(
+        twin0, yvals+0.075, yvals-0.1, edgecolor='none', color='darkorange')
+
+    # Automatically sets twinned axis colors
+    ex2.autocolor_spines()
```
Comment (mscross) [inline README.rst]: You cut off the remainder of the example.  The end result looks like this:
image

Comment (JDWarner) [inline README.rst]: I accidentally the whole ending. On it.

Comment (JDWarner) [inline README.rst]: Fixed! Data looks a little different thanks to the pseudorandom set seed, otherwise formatting looks right to me.

**Comment by JDWarner** [comment]
Corrected final example:
ex2

**Comment by mscross** [comment]
thank you!

**Pull request merged**